### PR TITLE
Cleanup and fix documentation

### PR DESCRIPTION
**Pull request by fiorix**
Update URLs to JSON schema, update/fix API documentation.

### CODE REVIEW

**Comment by skogtwin** [comment]
It is discouraged to merge one's own PRs... I'd merge it for you if you'd wait 5 minutes :)

File: vulndb/schema.go [hunk 1]
```diff
@@ -1,17 +1,3 @@
-// Copyright (c) Facebook, Inc. and its affiliates.
```
Comment (n1chre) [inline vulndb/schema.go]: Why did you remove this @fiorix ?

**Comment by fiorix** [comment]
@skogtwin merged it myself because there's no changes to code, only documentation.

@n1chre that file is auto-generated; went away on go generate but I've re-added in #54
